config/gatekeeper/apps_v1_deployment_gatekeeper-controller-manager.yaml

```diff
@@ -16,8 +16,6 @@ spec:
       gatekeeper.sh/system: "yes"
   template:
     metadata:
-      annotations:
-        container.seccomp.security.alpha.kubernetes.io/manager: runtime/default
```

Comment:
why this is removed

Reply:
This was removed in upstream Gatekeeper but I add it back using a Kustomize patch.